benches/PERFORMANCE.txt

```diff
@@ -0,0 +1,77 @@
+Machine d77ebe8481884bc3b2778c8083f1bf459e548e929edd87041beb14f6b868d35f:
+  Platform: macOS-13.1-arm64-arm-64bit
+  Timestamp: 2023-02-02 17:09:14.780857+00:00
+  AquaVM version: 0.35.0
+  Benches:
+    big_values_data: Loading a trace with huge values
+      air::preparation_step::preparation::prepare: 1.07s
+        air_interpreter_data::interpreter_data::serde_json::from_slice: 1.06s
+        air_parser::parser::air_parser::parse: 3.66ms
+        air::preparation_step::preparation::make_exec_ctx: 2.69ms
+      air::runner::execute: 246.00µs
+      air::farewell_step::outcome::from_success_result: 106.40ms
+        populate_outcome_from_contexts: 104.00ms
+          to_vec(data): 98.22ms
+          to_vec(call_results): 180.00µs
+    dashboard: big dashboard test
+      air::preparation_step::preparation::prepare: 53.07ms
+        air_interpreter_data::interpreter_data::serde_json::from_slice: 27.66ms
+        air_parser::parser::air_parser::parse: 12.23ms
+        air::preparation_step::preparation::make_exec_ctx: 3.11ms
+      air::runner::execute: 106.20ms
+        execute: 88.32ms
+          new: 53.74ms
+            resolve_ast_scalar: 39.13ms
+              resolve_ast_variable: 24.46ms
+                resolve_variable: 7.80ms
+          execute: 14.25ms
+            resolver::resolve::resolve_ast_variable: 1.29ms
+              resolve_variable: 350.00µs
+            instructions::call::resolved_call::prepare_request_params: 2.72ms
+              resolver::resolve::resolve_ast_variable: 418.00µs
+                resolve_variable: 134.00µs
+              instructions::call::resolved_call::serde_json::to_string(tetraplets): 789.00µs
+      air::farewell_step::outcome::from_success_result: 26.83ms
+        populate_outcome_from_contexts: 25.38ms
+          to_vec(data): 21.73ms
+          to_vec(call_results): 365.00µs
+    long_data: Long data trace
+      air::preparation_step::preparation::prepare: 310.00ms
+        air_interpreter_data::interpreter_data::serde_json::from_slice: 292.40ms
+        air_parser::parser::air_parser::parse: 3.73ms
+        air::preparation_step::preparation::make_exec_ctx: 3.39ms
+      air::runner::execute: 250.00µs
+      air::farewell_step::outcome::from_success_result: 29.03ms
+        populate_outcome_from_contexts: 26.65ms
+          to_vec(data): 21.49ms
+          to_vec(call_results): 182.00µs
+    network_explore: network_explore-inspired script
```

Comment:
let's just write smth like "a test where a N peers in a network are discovered" as a comment for this test